### ISSUE TITLE
Sprint review session - Sprint 1

### ISSUE THREAD

**Issue by writerly03** [hyphacoop/restructuring-futures#41]
Review criteria:

- what worked well?
- what did not work?
- did we meet the success criteria for this sprint?
- what are the key lessons from this sprint?

**Comment by tripledoublev** [comment]
Here are answers to 2 of these questions from a technical standpoint:

**What worked well?**
 - Testing features: I built web prototypes that delivered features that can write to a replica.
 
**What did not work?**
 - Syncing attachments between replicas to the share

**Comment by writerly03** [comment]
Recap - Sprint 1 (Jan 16-29)

**Overview** 

The development work has been divided into five sprints, each being two weeks long. There is a one week rest between each sprint. The first two weeks of work were focused on setting up the work cadence between Hypha and UKAI (e.g. establishing co-working sessions). We also spent time brainstorming the benefits and challenges of using Earthstar, a relatively new protocol and ultimately decided to try to use it as much as possible to form the basis of the next prototype.

The features that were the focus of the development work were file sharing and voice notes.

**What worked well?**

Testing features: Vincent built web prototypes that delivered features that can write to a replica.

**What did not work?**
Syncing attachments between replicas to the share was problematic. There were reliability problems with syncing.

**Did we meet the success criteria for this sprint?**

We did not define success criteria as much of the work was foundational. 

**What are the key lessons from this sprint?**
